### DISCUSSION TITLE
[Project Beta] Group by customized field

### DISCUSSION THREAD

**Discussion by zuccamia** [community/community#8232] [Projects and Issues] [answered]
I like that I can organize by tasks in sprints/iterations of 1-2 weeks in project by the Iteration field. I created a view with all tasks grouped by Iterations but it's being reset to a general list view every time I refresh the page or move to another view. 
I tried applying grouping from the command pallette as suggested in the [doc](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/customizing-your-project-views#grouping-by-field-values) but only group by Status was available. It's only noted that grouping are unavailable for titles, assignees, repositories or labels (no mention of other iteration or customized field).
Is there something I may be missing here to persist grouping for a view?

**Comment by micahalcorn** [comment]
I created a custom "Priority" field and would like to be able to group by this. Although, it also seems like priority should be a core feature of projects instead of something custom.

**Comment by azenMatt** [comment, marked as answer]
Hi @zuccamia - it sounds like you need to save the changes made to the view in order to persist them:

![image](https://user-images.githubusercontent.com/7584089/145182763-eb8d9bb2-c066-4203-be2d-b93e66b77e00.png)

The 'blue dot' lets you know if you have any changes that you can choose to persist or just keep as temporary. 

Does this help?

**Reply by zuccamia** [reply]
Yes, that was it! I need to click the little arrow again to see the changes need saving (the menu disappeared after I set the grouping criteria). Thanks for your answer @azenMatt !
